csharp/change-notes/2021-06-04-tuple-members.md

```diff
@@ -0,0 +1,2 @@
+lgtm,codescanning
+* Members extracted from `TupleTypes` have been fixed to be assigned to the underlying ``struct ValueTuple`N``.
```

Comment:
> ``struct ValueTuple`N``

This is looking odd, notice the stray **`**.
Or is this indeed intentional?

Reply:
Yep, that's intended. Dotnet fully qualified names use this notation for generic types. Search for `backtick` in https://docs.microsoft.com/en-us/dotnet/api/system.type.gettype?view=net-5.0.